ext/cld3/extconf.rb

```diff
@@ -17,7 +17,9 @@
 require "mkmf"
 
 # Check pkg-config first to inform the library is missing if so.
-pkg_config("protobuf")
+unless pkg_config("protobuf") || have_library("protobuf")
+  abort "Can't find protobuf\n"
```

Comment:
`Can't find` sounds a bit odd. Could you change to `protobuf not found` `Failed to locate protobuf`, or something sort of, as many other CLI tools do?